tests/test_string.py

```diff
@@ -17,12 +17,17 @@
 class StringTest(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
-        print 'connect to 127.0.0.1:7379\n'
-        cls.r = RedisWrapper('127.0.0.1', 7379).get_instance()
+        print 'connect to 127.0.0.1:5379\n'
+        cls.r = RedisWrapper('127.0.0.1', 5379).get_instance()
         cls.k1 = '__string1__'
         cls.v1 = 'value1'
         cls.k2 = '__string2__'
         cls.v2 = 'value2'
+	cls.bitKey = 'a'
```

Comment:
Here you can just use `cls.k1` or `cls.k2`, because unit test will clean after each test.